### PR TITLE
Fix error message when CAS already is in standby detach state

### LINKED ISSUE #1205: Misleading info after try detaching cache in standby detached state.
## Description
Misleading info after try detaching cache in standby detached state.

## Steps to Reproduce
1. Start standby cache instance
2. detach standby cache
3. check if cache is in standby detached state
4. try detach once more

## Logs
```
[root@localhost open-cas-linux]# casadm -L                                 
type    id   disk   status             write policy   device                                
cache   1    -      Standby detached   -              -                                               
[root@localhost open-cas-linux]# casadm --standby --detach -i 1
Failed to remove core. See dmesg for more information
```

### PR DESCRIPTION
**Pull request by pdebski21**
Fixes #1205

### CODE REVIEW

File: modules/include/cas_ioctl_codes.h [hunk 1]
```diff
@@ -557,6 +557,9 @@ enum kcas_error {
 	/** Removing core failed and rollback failed too */
 	KCAS_ERR_DETACHED,
 
+	/** Cache is already in standby detached state */
+	OCF_ERR_STANDBY_DETACHED,
```
Comment (jfckm) [inline modules/include/cas_ioctl_codes.h:561]: This is, clearly, a KCAS error code as it originates from adapter, not from OCF : `OCF_` -> `KCAS_`

**Review by jfckm — CHANGES_REQUESTED** [review]

File: modules/include/cas_ioctl_codes.h (hunk 1 above)
Comment (pdebski21) [reply modules/include/cas_ioctl_codes.h]: Yes, you are right. I removed it.

**Review by jfckm — DISMISSED** [review]

**Review by DocentSzachista — APPROVED** [review]

**Review by karolinavelkaja — DISMISSED** [review]

**Comment by karolinavelkaja** [comment]
retest this please

**Comment by karolinavelkaja** [comment]
retest this please

**Comment by jfckm** [comment]
@pdebski21 I believe it's complaining because `OCF_ERR_STANDBY_DETACHED` isn't present in `ocf_error_t`. I thought it was there and you were just reusing already existing error code - if that's not the case I guess you should put this error back in `enum kcas_error`, but this time with `KCAS_` prefix, not `OCF_`

**Review by jfckm — APPROVED** [review]

**Review by karolinavelkaja — APPROVED** [review]

**Pull request merged**